cmd/main.go

```diff
@@ -112,10 +115,10 @@ func runGrpcServer(grpcPort int, tlsFiles string, store gokv.Store) {
 	)
 	s := grpc.NewServer(serverOptions...)
 
-	bridgeServer := bridge.NewServer(store)
-	portServer := port.NewServer(store)
-	vrfServer := vrf.NewServer(store)
-	sviServer := svi.NewServer(store)
+	bridgeServer := bridge.NewServer(store, frrAddress)
```

Comment:
I prefer to inject dependency itself, rather than propagating a dependency parameter. If we need to add one more parameter, next time we do not pass it through the entire codebase, but set it in one place at wiring stage.

Reply:
I was thinking about it, we already have `NewServerWithArgs(store, nlink, frr)` so we can just use it...